src/super_context.ml

```diff
@@ -450,6 +213,8 @@ module External_env = Env
 
 module Env : sig
   val ocaml_flags : t -> dir:Path.t -> Ocaml_flags.t
+
+  val _external_ : t -> dir:Path.t -> External_env.t
```

Comment:
If this is temporary, don't forget to remove or rename it :)

Reply:
I actually planned to use this to fixed the bug. I suppose I can get rid of it for now.